teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './person_class'
+
+class Teacher < Person
+  attr_accessor :specialization, :id
+
+  def initialize(age, name, specialization, parent_permission)
+    @specialization = specialization
+    super(
+      age,
+    name,
+    parent_permission
+  )
```

Comment:
- [ ] Kindly reorder the position of the parameters because the parent class which is Person is taking **parent_permission** before the **name** 🤔 So kindly do something like this 👍 

⬇️
```
super(
  age,
  parent_permission,
  name
)
```

Reply:
@HAMMAS-SALEEM  please, take a view!